interfaces/bspline_vector_space.py

```diff
@@ -21,10 +18,10 @@ def __init__(self, degree=0, knots=[0., 1.]):
         self.mults = self.compute_mults(self.knots)
 
         assert ( self.n_knots - (self.degree + 1) ) > 0
-        self.n_dofs = self.n_knots - (self.degree + 1)
+        n_dofs = self.n_knots - (self.degree + 1)
```

Comment:
Più tardi nel codice viene richiamato il costruttore della classe padre e sarà lui a settare self.n_dofs nel modo che riterrà più opportuno. Al momento (e credo per sempre) il costruttore setta self.n_dofs=n_dofs ma, magari, dopodomani potrei aver voglia modificare questa cosa. Avrei potuto lasciare il self e poi il costruttore semplicemente eseguiva self.n_dofs=self.n_dofs ma volevo che fosse esplicito dal codice che self.n_dofs  non sarà settata in quel momento lì ma più tardi.